.npmignore

```diff
@@ -1,17 +1,20 @@
 debugging/
+docs/
+example/
```

Comment:
Let's keep the example so people have a tiny thing to run. It's not concerning in terms of file size.